pkg/label/label_selinux_test.go

```diff
@@ -28,13 +28,13 @@ import (
 func TestInit(t *testing.T) {
 	if selinux.SelinuxEnabled() {
 		var testNull []string
-		plabel, mlabel, err := InitLabels(testNull)
+		plabel, mlabel, err := InitLabels("", testNull)
```

Comment:
The tests should probably set some ephemeral test dir for mcsdir instead of "", but for correctness they should really be using a system-wide repository of these things.  I punted on it by leaving them as "" expecting review to trip on it, what do you guys think is most appropriate for the mcsdir in the tests? /cc @mjg59